src/instrumentation/express/error_handler.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response, NextFunction, ErrorRequestHandler } from "express"
```

Comment:
This branch branched of the OpenTelemetry branch before #715 was merged so this commit shouldn't be here. Can you rebase on the latest OpenTelemetry branch state?